index.d.ts

```diff
@@ -0,0 +1,20 @@
+declare module 'apollo-datasource-mongodb' {
+  import { DataSource, DataSourceConfig } from 'apollo-datasource'
+  import * as mongoose from 'mongoose'
+
+  export {
+    DataSourceConfig
+  }
+
+  interface Options { ttl: number }
+
+  export abstract class MongoDataSource<T extends mongoose.Document, TContext = any> extends DataSource {
+    collection: mongoose.Collection
+    model?: mongoose.Model<T>
+    constructor(collection: mongoose.Collection | mongoose.Model<T>)
+    public initialize(config: DataSourceConfig<TContext>): void
+    protected findOneById(id: string, { ttl }?: Options): T | null
+    protected findManyByIds(ids: string[], { ttl }?: Options): (T| null)[]
```

Comment:
The ids are supposed to be `ObjectId`s, at least for the native driver. Can you confirm @lorensr

Reply:
Yes. And if it comes in as a string argument, I use this function to convert:

![image](https://user-images.githubusercontent.com/251288/71549756-c8da4a80-2990-11ea-94fe-83b19063109c.png)